src/dmd/mars.d

```diff
@@ -2171,7 +2171,7 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
                     auto flag = tryParseJsonField(p + 4);
                     if (!flag)
                     {
-                        error("unknown JSON field `-Xi=%s`, expected one of " ~ jsonFieldNames, p + 4);
+                        error(cast(const(char)*)("unknown JSON field `-Xi=%s`, expected one of " ~ jsonFieldNames), p + 4);
```

Comment:
I haven't looked closely at the actual rewrite, but if it needs this additional cast (probably concatenation moved from CT to RT) it will break quite some code,